src/ui/pages/WalletSelect/WalletList.tsx

```diff
@@ -76,121 +86,154 @@ function WalletListItem({
+                            style={{
+                              display: 'inline-block',
+                              width: COPY_BUTTON_SIZE,
+                              height: COPY_BUTTON_SIZE,
+                              ['anchorName' as string]: anchorName,
+                              verticalAlign: 'bottom',
+                            }}
+                          ></span>
+                        ) : (
+                          copyButton
+                        )}
+                      </>
+                    )}
+                  />
+                </UIText>
+              }
+              detailText={
+                <PortfolioValue
+                  address={wallet.address}
+                  render={(query) => (
+                    <UIText kind="headline/h3">
+                      {query.data ? (
+                        <NeutralDecimals
+                          parts={formatCurrencyToParts(
+                            query.data.data?.totalValue || 0,
+                            'en',
+                            currency
+                          )}
+                        />
                       ) : (
-                        copyButton
+                        NBSP
                       )}
-                    </>
+                    </UIText>
                   )}
                 />
-              </UIText>
-            }
-            detailText={
-              <PortfolioValue
-                address={wallet.address}
-                render={(query) => (
-                  <UIText kind="headline/h3">
-                    {query.data ? (
-                      <NeutralDecimals
-                        parts={formatCurrencyToParts(
-                          query.data.data?.totalValue || 0,
-                          'en',
-                          currency
-                        )}
-                      />
-                    ) : (
-                      NBSP
-                    )}
-                  </UIText>
-                )}
-              />
-            }
-          />
-          {isSelected ? <CheckIcon style={{ width: 24, height: 24 }} /> : null}
-        </HStack>
+              }
+            />
+            {isSelected ? (
+              <CheckIcon style={{ width: 24, height: 24 }} />
+            ) : null}
+          </HStack>
+          {exploreRewardsUrl ? (
+            <Button
+              kind="neutral"
+              as={UnstyledAnchor}
+              href={exploreRewardsUrl}
+              target="_blank"
+              size={36}
+              style={{
+                borderRadius: '0 0 18px 18px',
+              }}
+            >
+              <HStack gap={8} alignItems="center" justifyContent="center">
+                <RewardsIcon
+                  style={{
+                    width: 20,
+                    height: 20,
+                    color: 'linear-gradient(90deg, #A024EF 0%, #FDBB6C 100%)',
```

Comment:
I believe this will be worth adding as a variable soon. Something like `--loyalty-gradient`

Reply:
these gradients are different everywhere... yeah, let's wait and see how many variations we'll end up with eventually :)